setup.cfg

```diff
@@ -0,0 +1,32 @@
+[metadata]
+name = ConfigArgParse
+version = attr: setuptools_git_versioning.version_from_git
+author = Ben Weisburd
+author_email = weisburd@broadinistitute.org
```

Comment:
would prefer not to list my email here

Reply:
Sure, done. It can still be dug out of the Git history but was already there in the Git logs.